### PR TITLE
Add a 'Table of Content' to the settingUp.md #5112

### LINKED ISSUE #1760: Incorrect error message when adding an existing instructor to a course
_From [OTS.Nexus](https://code.google.com/u/OTS.Nexus/) on March 29, 2014 03:38:42_

Current error message:
"An instructor with the same ID already exists in the course."

Since user doesn't key in the google ID anymore when adding new instructor, it should updated according.

**Attachment:** [incorrectErrorMessage.jpg](http://code.google.com/p/teammatespes/issues/detail?id=1763)

_Original issue: http://code.google.com/p/teammatespes/issues/detail?id=1763_

### LINKED ISSUE #5112: Add a 'Table of Content' to the settingUp.md
Add a hyperlinked table of content at the top of `/devdocs/settingUp.md`

### PR DESCRIPTION
**Pull request by krishnilak**
Fixes #5112

### CODE REVIEW

**Comment by LowWeiLin** [comment]
@krishnilak You are referencing the wrong issue in the pull request description. Please check again. Thanks!

**Comment by krishnilak** [comment]
@LowWeiLin I changed, Thank You

**Comment by LowWeiLin** [comment]
@krishnilak Sorry for the late reply. There seems to be changes unrelated to adding a table of contents. 
It is good that you can find improvements to be made to the document, but could you open a separate issue and pull request for them? We would like our pull requests to fix only the specified issue so that we can easily monitor the changes. 
Do remove the changes not required for adding the table of contents.
Thanks!

File: devdocs/settingUp.md [hunk 1]
```diff
@@ -22,10 +31,10 @@ Important: When a version is specified, please install that version instead of t
    Further instructions for installing can be found at https://developers.google.com/eclipse/docs/using_sdks.
 8. Install the latest [TestNG Eclipse plugin](http://testng.org/doc/download.html).
 
-##Setting up the dev server
+##<a name="setDevServer"></a>Setting up the dev server
```
Comment (damithc) [inline devdocs/settingUp.md]: See if you can avoid adding anchors. I think markdown supports linking to headings without adding anchor tags.

**Comment by damithc** [comment]
The unrelated changes may be coming from another PR? Perhaps this branch hasn't been synced properly?

**Comment by krishnilak** [comment]
@LowWeiLin Hi,I have committed changes again.

**Comment by wkurniawan07** [comment]
Hi @krishnilak @LowWeiLin, any progress on this PR? It'll be great if you can finish this soon. If you can't finish it by this milestone, let's push it to the next one. However, if you abandon it for too long, I'm afraid I have to close it.

**Comment by LowWeiLin** [comment]
@krishnilak The comment on html anchors has not yet been addressed, and there are merge conflicts to be resolved.

File: devdocs/settingUp.md [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+<<<<<<< HEAD
+>If you encounter any problems during the setting up process, please refer to our [troubleshooting guide](https://docs.google.com/document/d/1_p7WOGryOStPfTGA_ZifE1kVlskb1zfd3HZwc4lE4QQ/pub?embedded=true) before posting a help request in our [issue tracker](https://github.com/TEAMMATES/teammates/issues).
+
+These instructions are for the Windows environment. Instructions for Mac OS is similar, with slight variations that you can figure out yourself.
+=======
 >If you encounter any problems during the setting up process, please refer to our [troubleshooting guide](troubleshooting-guide.md) before posting a help request in our [issue tracker](https://github.com/TEAMMATES/teammates/issues).
 
 These instructions work for Linux, OS X as well as for the Windows
 environment. The only difference for Windows environment is that the command `./gradlew` should be replaced by `gradlew.bat` everywhere.
+>>>>>>> fbb2c78a651b240fb1e02f18a5b331f786be6ff4
```
Comment (wkurniawan07) [inline devdocs/settingUp.md]: You should resolve the conflicts properly instead of committing all these.

File: devdocs/settingUp.md [hunk 3]
```diff
@@ -48,6 +63,11 @@ Important: When a version is specified, please install that version instead of t
 3. Run the command `./gradlew setUp`.<br>
    This creates the main config files {These are not under revision control because their content vary from developer to developer}.
    * `src/main/resources/build.properties`<br>
+<<<<<<< HEAD
+   Use `build.template.properties` (in the same folder) 
+   as the template (i.e. `duplicate -> remove '.template' from name`).
+=======
+>>>>>>> fbb2c78a651b240fb1e02f18a5b331f786be6ff4
```
Comment (wkurniawan07) [inline devdocs/settingUp.md]: Similarly here.

**Comment by wkurniawan07** [comment]
Hi @krishnilak, let's try to finish this soon. If you can't finish it by this milestone, let's push it to the next one. However, if you abandon it for too long, I'm afraid I have to close it.

**Pull request closed without merging**